Assets/scripts/Enemy/Controller/ControllerRedEnemy.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ControllerRedEnemy 
```

Comment:
we don't need child classes for enemy controllers and models - custom data is controlled through different scriptable objects (SO)